src/handbook/peopleops/compensation.md

```diff
@@ -16,11 +16,31 @@ on compensation expectation of the candidate as well as the market rates. Salary
 is based on your location you live in and the company context. FlowForge pays in
 your local currency unless explicitly discussed otherwise.
 
+## Transition to Single-Point Increase Cycle and Inflation Correction
+
+We are making some changes to how we handle salary increases at FlowForge. We're moving away from the current two-point increase cycle and adopting a single-point increase cycle. This will make the process more streamlined and predictable for everyone. Along with this change, we will also be incorporating inflation correction for applicable countries. 
```

Comment:
```suggestion
We are making some changes to how we handle salary increases at FlowForge. We're moving away from the current two-point increase cycle and adopting a single-point increase cycle. This will make the process more streamlined and predictable for everyone. Along with this change, we will also be incorporating inflation correction for applicable countries. 
```

"We are making some changes" - That's the process before the PR is merged, and I think is a part of the PR description field. Not the handbook text